src/components/Item/components/Blessings.tsx

```diff
@@ -47,7 +47,7 @@ export function Blessings({ offer, items }: Props) {
 
       <div>
         {offer.description.overrides.traits.map((trait) => {
-          let { description: desc, display_name } = localisation[trait.id]
+          let { description: desc, display_name } = localisation[trait.id] || trait.id
```

Comment:
This will set `desc` & `display_name` to undefined when `localisation[trait.id]` does not exist. I suspect you intended to surface the `trait.id`.